cdap-master/src/main/java/co/cask/cdap/data/startup/BaseMasterCheck.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.data.startup;
```

Comment:
o didn't even realize this.  Everything in cdap-master is actually under co.cask.cdap.data, which doesn't seem right.  I can rename, but think its better to do it in a separate PR since it will pull in a lot more classes.